src/app/shared/store/registration.state.ts

```
@@ -192,33 +183,59 @@
     | Observable<Employee>
     | Observable<MinistryAdmin>
     | Observable<RegionAdmin>
-    | Observable<AreaAdmin> {
+    | Observable<AreaAdmin>
+    | void {
     const state = getState();
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)